### PR TITLE
Add PHP 8.0 CI + PHP 8.1 support

### PR DESCRIPTION
**Pull request by atomiix**
This PR aims to check the compatibility with PHP 8.0 and prepare the PHP 8.1 compatibility.

### CODE REVIEW

**Comment by Shudrum** [comment]
Hello! Thank you for your pull request.

Sorry, I have no idea why Travis status didn't showed up. [But it seems the tests are not passing](https://travis-ci.org/github/Shudrum/ArrayFinder/requests):

image

**Comment by atomiix** [comment]
Hey @Shudrum, I don't know what happened either but it looks like tests are not executed at all, the job has been "rejected" 🤷‍♂️

**Comment by carusogabriel** [comment]
I have a feeling this is related to GitHub's change for first-time contributors: https://github.blog/changelog/2021-04-22-github-actions-maintainers-must-approve-first-time-contributor-workflow-runs

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,8 @@ matrix:
       dist: bionic
     - php: 7.4
       dist: bionic
+    - php: 8.0
+      dist: bionic
```
Comment (carusogabriel) [inline .travis.yml:22]: @atomiix Would you mind adding PHP 8.1 to the build as well?

**Comment by rdohms** [comment]
bump

**Comment by ricardofontanelli** [comment]
Hey @atomiix and @Shudrum do you need any help to move this PR forward? Would be awesome to release support to 8.1 and I think we're pretty close on this PR. Anything that I can help with?

**Comment by atomiix** [comment]
@Shudrum from what I understand it looks like we can't use travis-ci.org anymore, travis-ci.com should be used: https://docs.travis-ci.com/user/migrate/open-source-repository-migration#what-will-happen-to-my-travis-ciorg-repository

**Comment by jcmiller09** [comment]
@atomiix @Shudrum, Can we get the php 8.1 version updates into a new PR?

**Pull request closed without merging**